MetaMask/metamask-extension
# refactor: move ConfirmInfoRow files out of shared directory

## **Description**

After recent discussion and alignment, we will no longer need the "shared" directory. This PR moves the ConfirmInfoRow components one nest closer

## **Related issues**

fixes: https://github.com/MetaMask/metamask-extension/pull/19957
fixes: https://github.com/MetaMask/metamask-extension/issues/21751

## **Manual testing steps**

Verify `ui/components/app/snaps/snap-ui-renderer/snap-ui-renderer.js` works as expected

Verify components work as expected 
see e.g.
1. yarn storybook
2. http://localhost:6006/?path=/docs/components-app-confirm-inforowvaluedouble--docs


## **Screenshots/Recordings**



### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
